### PR TITLE
Add klipfolio datasources monitoring.

### LINKED ISSUE #17: add monitoring of data sources
Send out an alert if a Klipfolio data source does not refresh in a specified amount of time or fails to refresh.

### PR DESCRIPTION
**Pull request by benkags**
closes #17 

cc @henokgetachew

### CODE REVIEW

**Comment by benkags** [comment]
Would love have to have this deployed as soon as possible. cc @henokgetachew @medic/site-reliability-engineering.

**Comment by derickl** [comment]
@medic/site-reliability-engineering 

Could you please take a look at this?

**Comment by benkags** [comment]
Any review / update here ?

File: plugins/config.json [hunk 1]
```diff
@@ -78,5 +78,13 @@
     "schedule": "60000",
     "servers": "all",
     "active": true
+  },
+  {
+    "name": "Klipfolio datasources monitoring",
+    "description": "Ensures specified datasources are refreshing",
+    "path": "./plugins/scripts/klipfolio_datasources_test.js",
+    "schedule": "12000",
```
[the remark this answers is not in the document]
Comment (benkags) [reply plugins/config.json]: Right. Changing to 12 hrs which was the intended frequency.

[the remark this answers is not in the document]
Comment (benkags) [reply plugins/config.json]: Done

File: plugins/data_sources.json.example [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    {
+        "name": "LG Innovation Network's Data sources",
+        "user": "test@user.com",
+        "password": "pass",
+        "datasources_uuids": [
+            "27075d0a083edc4aa310e4c8f6216781",
+            "127075d0a083edc4aa310e4c8f621697"
+        ]
+    }
+]
```
[the remark this answers is not in the document]
Comment (benkags) [reply plugins/data_sources.json.example]: I miscommunicated about pwd-file. My bad. The implementation does not require it. User credentials specified in `klipfolio/data_sources.json` are used instead.

**Pull request merged**